### ISSUE TITLE
warning: URI.unescape is obsolete

### ISSUE THREAD

**Issue by coding-bunny** [savonrb/wasabi#76]
Hello,

Running `wasabi` 3.5.0 under Ruby 2.7 and my test-suite is being spammed with these messages:

```
/Users/olivar/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/wasabi-3.5.0/lib/wasabi/parser.rb:89: warning: URI.unescape is obsolete
/Users/olivar/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/wasabi-3.5.0/lib/wasabi/parser.rb:90: warning: URI.escape is obsolete
```

This is a minor thing, but would it be possible to have these resolved in the next version?

**Comment by walamsyah** [comment]
It's already on PR https://github.com/savonrb/wasabi/pull/75.

For a while you can update your Gemfile with:
`gem 'wasabi', github: 'bvicenzo/wasabi', ref: 'ba481c54dc40da395bc862aa11d7f48a1733d935'`

**Comment by coding-bunny** [comment]
Thanks, I'll bear with it for now till the new version that will fix it. Just happy it's being resolved :)

**Comment by Louis-T** [comment]
I know the PR will be merged soon.
But in the meantime, if someone has a problem with malformed URI in savon/wasabi, you juste have to update the ref: `ref: 'abc869f4c80d15b5cf527cc8a938732a5b7bb94d'`.

The `ba481c54dc40da395bc862aa11d7f48a1733d935` ref was a little buggy !

**Comment by olleolleolle** [comment]
Fixed in #81 which was extracted from #75. Thanks!
